cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -315,12 +315,26 @@ def _add_resources(self):
                 cw_log_group=self.log_group,
             )
 
-            self._add_alarms()
+            self._add_head_node_alarms()
 
-    def _add_alarms(self):
-        self.alarms = []
+    def _add_head_node_alarms(self):
+        self.head_node_alarms = []
 
         metrics_for_alarms = {
+            "Health": cloudwatch.Metric(
```

Comment:
Did you evaluate the possibility to create an internal function to avoid code replication? period, statistic, dimension_map, namespace could have default values and simplify creation of new metrics.

Reply:
done